### PR TITLE
Added JSONBuddy Cli tool

### PR DESCRIPTION
**Pull request by Clemens-U**

### CODE REVIEW

File: _data/validator-libraries-modern.yml [hunk 1]
```diff
@@ -362,3 +362,9 @@
       date-draft:
       draft: [7, 6, 4]
       notes: wraps [xeipuuv/gojsonschema](https://github.com/xeipuuv/gojsonschema)
+    - name: valbuddy
+      license: Free and commercial versions
```
Comment (Relequestual) [inline _data/validator-libraries-modern.yml:366]:
```suggestion
      license: Free and commercial versions (proprietary)
```

**Review by Relequestual — CHANGES_REQUESTED** [review]
I suspect people will want to know if you're using a standard open source license or not. The suggestion should mak that clearer.
Thanks for adding! =]

**Comment by Clemens-U** [comment]
Is there anything open from my side because of the requested change?

**Review by Relequestual — APPROVED** [review]

**Comment by Relequestual** [comment]
Sorry for the delay. Thanks again for your work!

**Comment by handrews** [comment]
Not sure why this wasn't merged.  @gregsdennis you might need to update your most recent PR, but I don't want to ask this person to change theirs a month after approval.

**Pull request merged**